source/schema/export.txt

```diff
@@ -24,156 +24,95 @@ If you have not already done so, analyze your schema:
    subset of documents from your collection. To learn more about 
    sampling, see :ref:`Sampling <compass-sampling>`.
 
-Once your schema has been analyzed, export your schema:
+Once your schema has been analyzed, export your schema. 
 
-1. In the top menu bar, click :guilabel:`Collection`.
+1. Click :guilabel:`Export Schema`.
```

Comment:
[question] Should we use the procedure directive here?

Reply:
Yes! I've updated several procedures in this file to use the `.. procedure::` directive. Thanks!

Reply:
Yes! I've updated several procedures in this file to use the `.. procedure::` directive. Thanks!